vitessce/data_utils/multivec.py

```diff
@@ -84,20 +112,27 @@ def convert_bin_name_to_chr_end(bin_name):
     )
     chrom_name_to_length = genomic_profiles.chrom_name_to_length
 
+    # For debugging purposes, allow the user to specify a subset of chromosomes to process.
+    chrom_names = chr_subset if chr_subset is not None else list(chrom_name_to_length.keys())
+
     # Create each chromosome dataset.
-    for chr_name, chr_len in chrom_name_to_length.items():
+    for chr_name in tqdm(chrom_names):
+        assert chr_name in chrom_name_to_length
```

Comment:
This would only fail if the `chr_subset` member is not specified in the given genomic assembly, so it's probably safe, but I figure it's worth double checking/asking about since I've never seen an assert statement outside of testing contexts.

Reply:
Good point, I have removed the assert. Should result in a KeyError on the subsequent line anyway